lib/template.html

```diff
@@ -131,18 +151,27 @@
   <meta property="og:site_name" content="{{ seo_site_title }}" />
 {% endif %}
 
-{% if seo_page_image %}
-<meta property="og:image" content="{{ seo_page_image }}" />
-  {% if page.image.height %}
-    <meta property="og:image:height" content="{{ page.image.height }}" />
+{% if page.image %}
+  {% if page.image.facebook %}
+    {% assign local_seo_page_image = seo_page_image_facebook %}
+    {% assign local_seo_page_image_height = seo_page_image_facebook_height %}
+    {% assign local_seo_page_image_width = seo_page_image_facebook_width %}
+  {% else %}
+    {% assign local_seo_page_image = seo_page_image_default %}
```

Comment:
I'm a little confused what `local_` means here. Is it a temp variable, or does it represent an image set in content front-matter (versus site config)? In either case, it feels funky to assign a _default_ to something that says `local_` sitting under branch logic which says `facebook`. Make sense?